app/styles/map-view.scss

```diff
@@ -1,6 +1,6 @@
 .map-view {
   padding: 0;
-  height: calc(100vh - 226px);
+  height: calc(100vh - 82px);
```

Comment:
not sure whether you originated the hight offset, but whenever I encounter magic numbers like these I keep my fingers crossed for some comment nearby to help me understand them ....